portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -64,11 +65,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = getParameter(request, "text-input", "");
     long timestamp = System.currentTimeMillis();
+
+    SimpleDateFormat sdf = new SimpleDateFormat("yyyy.MM.dd G 'at' HH:mm:ss z");   
     
 
     Entity messageEntity = new Entity("Message");
     messageEntity.setProperty("text", text);
-    messageEntity.setProperty("timestamp", timestamp);
+    messageEntity.setProperty("timestamp", sdf.format(timestamp));
```

Comment:
Woah, you just taught me something. I assumed the `format()` function needed a `Date` instance. I didn't realize it worked with `long` primitives too. Cool!